tests/commands/run_test.py

```diff
@@ -1,4 +1,4 @@
-# -*- coding: UTF-8 -*-
+# -*- coding: utf-8 -*-
```

Comment:
I forget where I saw the benchmark but the basic reason is when looking up the codecs it tries the value first and then normalizes it, the speed difference is negligible, but changing this line is unnecessary to your patch